software/tine.yml

```
@@ -0,0 +1,10 @@
+name: Tine
+website_url: https://www.tine-groupware.de/
+description: tine is the ideal software for digital collaboration in companies and organizations. From powerful groupware functionalities to clever add-ons, tine combines everything to make daily team collaboration easier.
+licenses:
+  - AGPL-3.0
+platforms:
+  - PHP
```

```suggestion
  - Docker
```
[Installation Guide](https://tine-docu.s3web.rz1.metaways.net/operators/Installation_Guide/) only mentions Docker

---

Thanks.